c2corg_api/models/outing.py

```diff
@@ -78,7 +78,33 @@ class _OutingMixin(object):
 
     disable_comments = Column(Boolean)
 
-attributes = [
+    # ratings
+    # hiking
+    hiking_rating = Column(enums.hiking_rating)
```

Comment:
Please use blank lines as for the other properties above. I don't think the comments are really needed.

Reply:
wilco